### ISSUE TITLE
FR: Add an explanation file for each topic.

### ISSUE THREAD

**Issue by jonasstein** [mm1ke/gentoo-scripts#20]
I suggest to add a file with more information on the problem.
I am not sure, if it is better to put/link one file in each folder, or maintain one central file. 
It could be also a page in the personal user space of wiki.gentoo.org 
or markdown on github? 
I think it is important to add links there and that many users can contribute to the explanations.

The content should be a short explanation for the developer
_Why, is this a problem? Where is it documented? How to fix it?_

I think this would motivate more people to fix things...

```
000-README.txt or
000-README.md or
000-EXPLANATION.txt
```

**Comment by mm1ke** [comment]
Yes, i think a github/wiki page would be best. It's easier to read than a regular textfile.

**Comment by simonvanderveldt** [comment]
A wiki would probably work pretty well for this. For an example look at pylint's descriptions for it's warnings http://pylint-messages.wikidot.com/messages:c0103

**Comment by jonasstein** [comment]
Yes that looks great. Can we use the github wiki/website function for this? I have no experience with it, but I guess it can do markdown and we need no new accounts and everything is at one spot.

**Comment by simonvanderveldt** [comment]
> Yes that looks great. Can we use the github wiki/website function for this? I have no experience with it, but I guess it can do markdown and we need no new accounts and everything is at one spot.

Yeah, the wiki on GitHub is editable by everyone. It can also be set to be editable by contributors only.
